### DISCUSSION TITLE
anyone know a way of hiding files with e.g. no file-extention or *.thumbnails?

### DISCUSSION THREAD

**Discussion by mschuerewegen** [koreader/koreader#9399] [Q&A]
anyone know a way of hiding files with e.g. no file-extention or *.thumbnails?

i have a remarkable 2

and the default xochitl directory also houses thumbnails and meta fails.

i would like them hidden in the filebrowser.

**Comment by Frenzie** [comment]
Are you talking about some program on the Remarkable? I don't think KOReader should show *.thumbnail unless you enabled showing unsupported files.

**Reply by mschuerewegen** [reply]
These file show up in the koreader’s file browser.  I would like to hide them

**Reply by Frenzie** [reply]
That means that you enabled it in file browser (top left) → settings → show unsupported files.

**Reply by Toby1091** [reply]
@mschuerewegen Does that mean that you got Koreader running on a remarkable 2? Any chance you could share an instruction? The stuff I found online so far didn't work for me...

**Reply by NiLuJe** [reply]
> @mschuerewegen Does that mean that you got Koreader running on a remarkable 2? Any chance you could share an instruction? The stuff I found online so far didn't work for me...

https://github.com/koreader/koreader/wiki/Installation-on-ReMarkable
